pygeos/creation.py

```diff
@@ -170,3 +172,38 @@ def geometrycollections(geometries):
         An array of geometries
     """
     return lib.create_collection(geometries, GeometryType.GEOMETRYCOLLECTION)
+
+
+def prepare(geometry, **kwargs):
+    """Compute a prepared geometry
+
+    Prepared geometries improves performance of the binary predicate operations
+    contains, contains_properly, covered_by, covers, crosses, disjoint, intersects,
+    overlaps, touches, and within.
+
+    Parameters
+    ----------
+    geometry : Geometry or array_like
+
+    See also
+    --------
+    destroy_prepared
+    """
+    return lib.prepare(geometry, **kwargs)
+
+
+def destroy_prepared(geometry, **kwargs):
+    """Destroy a previously prepared geometry, freeing up memory.
+
+    Note that the prepared geometry will always be cleaned up if the geometry itself
+    is dereferenced. This function needs only be called in very specific circumstances.
```

Comment:
Should there be more guidance here or in docs about what those specific circumstances are?